packages/frontend-2/components/viewer/Controls.vue

```diff
@@ -396,30 +396,31 @@ const toggleActiveControl = (control: ActiveControl) => {
   activeControl.value = activeControl.value === control ? 'none' : control
 }
 
-onKeyboardShortcut([ModifierKeys.AltOrOpt], 'm', () => {
+onKeyboardShortcut([ModifierKeys.Shift], 'm', () => {
```

Comment:
Mapping between Keyboard Shortcut and expected action is duplicated, and thus its easy to make the mistake of setting one thing above and different one below, especially when there's so many actions. That may already be the reason behind Shift+Space not working, I notice its ' ' not 'Space' below, but that's just a guess.

Create a DRY mapping kinda like this:
```
enum ViewerKeyboardActions {
  ToggleModels,
  ToggleExplorer,
...
}

const map: Record<ViewerKeyboardActions, Array<ModifierKeys[], string>> = {
  [ViewerKeyboardActions.ToggleModels]: [[ModifierKeys.Shift], 'x'],
...
}

const modelsShortcut = ref(
  `Models (${getKeyboardShortcutTitle(map[ViewerKeyboardActions.ToggleModels])})`
)

onKeyboardShortcut(...map[ViewerKeyboardActions.ToggleModels], () => {
  toggleActiveControl('models')
})
```